velox/exec/Task.cpp

```diff
@@ -688,6 +710,16 @@ void Task::start(uint32_t maxDrivers, uint32_t concurrentSplitGroups) {
   }
 }
 
+void Task::checkExecutionMode(ExecutionMode mode) {
+  VELOX_CHECK_EQ(
+      mode,
+      mode_,
+      "Inconsistent task execution mode. Cannot execute in '{}' mode "
```

Comment:
You don't need to log the modes in check failure message as VELOX_CHECK_EQ will log out the mismatch modes.